handbook/ce/customer_issues.md

```diff
@@ -0,0 +1,32 @@
+# Filing customer issues
+
+Read [the support overview](index.md) before filing an issue.
+
+## Create a customer issue
+
+Customer support tickets should be translated to GitHub issues when they require an engineering team to be involved. In such cases, [create a
+new issue for the request](https://github.com/sourcegraph/customer/issues/new).
+
+Provide the appropriate context and add a label with the affected customer as `customer/$name`. Once its created, sharing it with the required [team](routing_questions.md).
+If necessary, link to the appropriate JIRA Service Desk ticket or [HubSpot](#find-the-unique-company-url) notes.
+
+### General issues
```

Comment:
This section describes to me a manual process for overcoming the underlying problem, which is having multiple sources of truth for issue tracking. See my proposal above.